stackdriver/client_test.go

```diff
@@ -120,7 +120,7 @@ func TestEmptyRequest(t *testing.T) {
 }
 
 func TestResolver(t *testing.T) {
-	addressesToTest := []string{"stackdriver.invalid", "2001:db8::"}
+	addressesToTest := []string{"stackdriver.invalid", "2001:db8:"}
```

Comment:
`2001:db8:` is not a valid ipv6.

Reply:
Reverted, will squash and merge and ignore the travis error and merge.